src/py42/clients/securitydata.py

```diff
@@ -14,13 +11,11 @@
 class SecurityDataClient:
     def __init__(
         self,
-        security_service,
         file_event_service,
         preservation_data_service,
         saved_search_service,
         storage_service_factory,
     ):
-        self._security_service = security_service
         self._file_event_service = file_event_service
         self._preservation_data_service = preservation_data_service
         self._saved_search_service = saved_search_service
```

Comment:
Lines 23 and 24 can be removed (as well as the `Lock` import),
it does not seem like they are used anymore